src/ape/__init__.py

```diff
@@ -3,96 +3,51 @@
 signal.signal(signal.SIGINT, lambda s, f: _sys.exit(130))
 
 import sys as _sys
-from functools import partial as _partial
-from pathlib import Path as _Path
+from functools import partial
+
+from ape.utils import ManagerAccessBase as _ManagerAccessBase
 
 from .contracts import _Contract
-from .managers.accounts import AccountManager as _AccountManager
-from .managers.chain import ChainManager as _ChainManager
-from .managers.compilers import CompilerManager as _CompilerManager
-from .managers.config import ConfigManager as _ConfigManager
-from .managers.converters import ConversionManager as _ConversionManager
-from .managers.networks import NetworkManager as _NetworkManager
 from .managers.project import ProjectManager as Project
-from .managers.project import _DependencyManager
-from .plugins import PluginManager as _PluginManager
-from .utils import USER_AGENT
 
 # Wiring together the application
-_data_folder = _Path.home().joinpath(".ape")
 
-plugin_manager = _PluginManager()
+plugin_manager = _ManagerAccessBase.plugin_manager
 """Manages plugins for the current project. See :class:`ape.plugins.PluginManager`."""
 
-_DependencyManager.plugin_manager = plugin_manager
-_dependency_manager = _DependencyManager(data_folder=_data_folder)
-
-_ConfigManager.plugin_manager = plugin_manager
-_ConfigManager._dependency_manager = _dependency_manager
-config = _ConfigManager(
-    # Store all globally-cached files
-    data_folder=_data_folder,
-    # NOTE: For all HTTP requests we make
-    request_header={
-        "User-Agent": USER_AGENT,
-    },
-    # What we are considering to be the starting project directory
-    project_folder=_Path.cwd(),
-)
-"""The active configs for the current project. See :class:`ape.managers.config.ConfigManager`."""
+config = _ManagerAccessBase.config_manager
+"""The active configs for the current project. See :class:`ape.managers.config.ConfigManager`.
+# NOTE: config is an injected property.
```

Comment:
This comment is not really a comment, it is embedded into the doc str
<img width="708" alt="Screen Shot 2022-02-25 at 09 43 49" src="https://user-images.githubusercontent.com/19540978/155744329-ae47c119-03f1-4d5c-890e-4f229c33d3b7.png">
n

Is this intentional? I don't think "injected property" means a whole lot to the user viewing this particular doc...

Reply:
I'll remove that, not as relevant now that we've changed some stuff around.